src/components/SalesChannel.js

```diff
@@ -0,0 +1,321 @@
+        }, [ecomdashStatus.data?.isInstalling])
+      
+      useEffect(() => {
+        let pluginConnectionStatus = async () => {
+            try {
+              const response = await fetch("/wp-admin/admin.php?page=newfold-ecomdash-settings");
+              const text = await response.text();
+              const parser = new DOMParser();
+              const doc = parser.parseFromString(text, 'text/html');
+              const element = doc.querySelector('#disconnect-instance');
+              setEcomdashSetupStatus(element === null);
+              return element === null 
+            } catch (error) {
+              console.error('Error fetching HTML:', error);
+            }
+        } 
+        if(ecomdashStatus?.data?.isInstalled){
+            pluginConnectionStatus()
+        }        
+      }, [ecomdashStatus?.data?.isInstalled])
+        
+    return(
+
+        hasYithExtended && canAccessGlobalCTB && hasEcomdash && (ecomdashStatus?.data?.isInstalled || ecomdashStatus?.data?.isNeedToInstall || ecomdashStatus.data?.isInstalling) ?
+        <>
+            <Section.Container>
+                <Section.Header title={__("Sales Channels", "wp-module-ecommerce")} subTitle={__("Sell your products everywhere, confidently, with Ecomdash.", "wp-module-ecommerce")} />
+                <Section.Content>
+                    
+                    <div className="nfd-flex  nfd-flex-row nfd-flex-wrap nfd-gap-6">                                             
+                       <div className="nfd-flex-1">
+                            <h2 className="nfd-text-[#0F172A] nfd-text-lg nfd-font-medium nfd-pb-1.5">
+                                    {__("Sell your products everywhere, confidently, with Ecomdash.", "wp-module-ecommerce")}
+                            </h2>  
+                            <p className="nfd-text-[#4A5567] nfd-pb-7 nfd-leading-5  nfd-font-medium">
+                                {__("With ecomdash, you can manage different sales channels all in one place.", "wp-module-ecommerce")}
+                                <br />
+                                {__("Simply connect your business accounts from Amazon, Etsy or Walmart to", "wp-module-ecommerce")}
+                                <br />
+                                {__("your ecomdash platform and easily manage products, customer orders and", "wp-module-ecommerce")}
+                                <br />
+                                {__("more saving you valuable time.", "wp-module-ecommerce")}​
+                            </p>
+                            <Button
+                                    className="nfd-button nfd-button--primary"
+                                    variant="primary" 
+                                    type="button"                                    
+                                    as={ canAccessGlobalCTB ?  hasEcomdash ? ecomdashStatus.data.isInstalled ? "a" : "button" : "button" : null }
```

Comment:
can we make it this way for better readability 
```suggestion
                                 as={  canAccessGlobalCTB ? ( hasEcomdash ? ( ecomdashStatus.data.isInstalled ? "a" : "button" ) : "button" ) : null }
```